src/install/kafka/default-configuration.mdx

```diff
@@ -0,0 +1,48 @@
+---
+componentType: default
+headingText: Configure the Kafka integration
+---
+
+1. Change directory to the integrations configuration folder by running:
```

Comment:
Should we be using the step component instead of steps here? Just a curiosity, I can see it maybe being too visually busy for this type of doc, so if you've tested it out already feel free to ignore.